src/Features/LanguageServer/Protocol/Handler/Initialize/InitializeHandler.cs

```diff
@@ -53,7 +53,11 @@ public Task<LSP.InitializeResult> HandleRequestAsync(Solution solution, LSP.Init
                     DocumentOnTypeFormattingProvider = new LSP.DocumentOnTypeFormattingOptions { FirstTriggerCharacter = "}", MoreTriggerCharacter = new[] { ";", "\n" } },
                     DocumentHighlightProvider = true,
                     ReferencesProvider = true,
-                    ProjectContextProvider = true
+                    ProjectContextProvider = true,
+                    TextDocumentSync = new LSP.TextDocumentSyncOptions
+                    {
+                        Change = LSP.TextDocumentSyncKind.None
```

Comment:
the default values for the other sync options are false, so they don't need to be explicitly set.  The capabilities previously were true because we returned a null TextDocumentSync, which then used the default value - https://devdiv.visualstudio.com/DevDiv/_git/VSLanguageServerClient?path=%2Fsrc%2Fproduct%2FRemoteLanguage.Protocol%2FServerCapabilities.cs&version=GBdevelop&line=28&lineEnd=29&lineStartColumn=1&lineEndColumn=1&lineStyle=plain )